src/igvfd/types/publication.py

```diff
@@ -0,0 +1,36 @@
+from snovault import (
+    collection,
+    load_schema,
+    calculated_property
+)
+
+from .base import (
+    Item,
+    datetime
+)
+
+
+@collection(
+    name='publications',
+    unique_key='publication:identifier',
+    properties={
+        'title': 'Publications',
+        'description': 'Publication pages',
```

Comment:
I don't think the description should be "Publication pages", should not it be "Listing of publications"?